composer.json

```diff
@@ -248,13 +259,14 @@
             "symfony/phpunit-bridge": true
         },
         "platform": {
-            "php": "8.1.0"
```

Comment:
NIT: This fixes notices when we run composer install saying that we have incompatible PHP version (8.1.X°